docs/source/ops/connect/auth0.rst

```diff
@@ -0,0 +1,657 @@
+APIs page from the menu on the left and click Create API in the top right.
+
+You can choose any name for the API; for the purposes of this document, we'll
+assume this API is named ``API_NAME``. The other parameters, however, are not
+free to set: the API identifier **has to be** ``https://daml.com/ledger-api``,
+and the signing algorithm **has to be** RS256 (which should be selected by
+default). Creating the API should automatically create a Machine-to-Machine
+application "API_NAME (Test Application)", which we will be using to generate
+our tokens. You can change its name to a more appropriate one; for the
+remainder of this document, we will assume it is called ADMIN_TOKEN_APP.
+
+Navigate to that application's settings page (menu on the left: Applications >
+Applications page, then click on the application's name). This is where you can
+rename the application and find out about its Client ID and Client Secret,
+which we'll need later on.
+
+Now that we have an API and an application, we can generate a token with the
+appropriate claims. In order to do that, we need to make an Auth0 Action.
+
+In the menu on the left, navigate to Actions > Library, then click on Build
+Custom in the top right. You can choose an appropriate name for your action;
+we'll call it ADMIN_TOKEN_ACTION. Set the Trigger field to
+"M2M/Client-Credentials", and leave the version of Node to the recommended one.
+(These instructions have been tested with Node 16.)
+
+This will open a text editor where you can add JavaScript code that will
+trigger on M2M (machine to machine) connections. Replace the entire text box
+content with:
+
+.. code-block:: javascript
+
+   exports.onExecuteCredentialsExchange = async (event, api) => {
+     if (event.client.client_id === "%%ADMIN_TOKEN_ID%%") {
+       api.accessToken.setCustomClaim(
+         "https://daml.com/ledger-api",
+         {
+           "ledgerId": "%%LEDGER_ID%%",
+           "participantId": null,
+           "applicationId": "party-creation",
+           "admin": true,
+           "actAs": []
+         }
+       );
+     }
+   };
+
+You need to replace ``%%ADMIN_TOKEN_ID%%`` with the Client ID of the
+ADMIN_TOKEN_APP application, and ``%%LEDGER_ID%%`` with your actual
+``ledgerId`` value. You can freely choose the ``applicationId`` value, and
+should set an appropriate ``participantId`` if your Daml driver requires it.
+
+You then need to click on Deploy in the top right to save this Action. Despite
+the text on the button, this does not (yet) deploy it anywhere.
+
+In order to actually deploy it, we need to make that Action part of a Flow. In
+the menu on the left, navigate through Actions > Flows, then choose Machine to
+Machine. Drag the "ADMIN_TOKEN_ACTION" box on the right in-between the "Start"
+and "Complete" black circles in the middle. Click Apply. Now your Action is
+"deployed" and, should you modify it, clicking on the Deploy button *would*
+directly affect your live setup.
+
```

Comment:
Would be nice to have one image showing the GUI. But that is really only nice to have.

Reply:
The GUI is exactly the same as all the way through the GSG, so I'm not sure how much value that adds. On the plus side, I suppose I could just put a link to the same image... 👿